presto-geospatial-toolkit/src/main/java/com/facebook/presto/geospatial/serde/GeometrySerde.java

```diff
@@ -67,14 +68,38 @@ public static Slice serialize(Envelope envelope)
         requireNonNull(envelope, "envelope is null");
         verify(!envelope.isEmpty());
         DynamicSliceOutput output = new DynamicSliceOutput(100);
-        output.appendByte(GeometryType.ENVELOPE.code());
+        output.appendByte(GeometrySerializationType.ENVELOPE.code());
         output.appendDouble(envelope.getXMin());
         output.appendDouble(envelope.getYMin());
         output.appendDouble(envelope.getXMax());
         output.appendDouble(envelope.getYMax());
         return output.slice();
     }
 
+    public static GeometryType getGeometryType(Slice shape)
```

Comment:
This function doesn't seem to be used anywhere.

Reply:
It gets used here: https://github.com/prestodb/presto/pull/10387/commits/1f5762da66c4ce927824b4fba338f89c6cc845c0#diff-8d5e0df6320e36d85d89168af14128f6R723

Reply:
@jwass I see. Then, perhaps, implementation should be `return deserializeType(shape).geometryType()`

Reply:
Oops. Yeah, much better. Fixed now.